packages/datadog-instrumentations/src/http/server.js

```diff
@@ -0,0 +1,71 @@
+'use strict'
+
+const {
+  channel,
+  addHook,
+  AsyncResource
+} = require('../helpers/instrument')
+const shimmer = require('../../../datadog-shimmer')
+const web = require('../../../dd-trace/src/plugins/util/web')
+
+const startServerCh = channel('apm:http:server:request:start')
+const endServerCh = channel('apm:http:server:request:end')
+const errorServerCh = channel('apm:http:server:request:error')
+const asyncEndServerCh = channel('apm:http:server:request:async-end')
+
+// wrap emitter of either the request, and response object
```

Comment:
This comment doesn't reflect what's actually happening. What you're wrapping, in this file anyway, is the `emit` method of the `Server` prototype. That's fairly obvious from the code though, so I don't think this comment is necessary at all.